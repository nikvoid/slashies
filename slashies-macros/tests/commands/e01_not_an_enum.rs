use slashies_macros::Commands;

#[derive(Commands)]
struct BadCommands {}

fn main() {}
