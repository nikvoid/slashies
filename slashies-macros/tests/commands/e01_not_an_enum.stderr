error: Can only derive Commands for enums
 --> tests/commands/e01_not_an_enum.rs:4:8
  |
4 | struct BadCommands {}
  |        ^^^^^^^^^^^
